docs/experimental.md

```diff
@@ -2,6 +2,8 @@
 
 In addition to our standard CodeQL queries and libraries, this repository may also contain queries and libraries of a more experimental nature. Experimental queries and libraries can be improved incrementally and may eventually reach a sufficient maturity to be included in our standard libraries and queries.
 
+Experimental queries and libraries may not be actively maintained as the standard libraries evolve, may be changed in backwards-incompatible ways or may be removed entirely in the future without deprecation warnings.
```

Comment:
This is rather long. Suggest simplifying it as follows but otherwise, please breakdown into shorter sentences.

```suggestion
Experimental queries and libraries are not actively maintained and may stop working as the standard libraries evolve.
```

I'd suggest omitting the extra information about backwards-compatibility, deprecation, and removal because there is no commitment given for this in the main contributing guidelines. Possibly those guidelines should be updated and then this section could state that, unlike the standard queries and libraries, there is no commitment to maintain the experimental files. The standard contributing document would explain what that maintenance involved.

Reply:
Happy to simplify as suggested.

@hvitved / @jbj: you had suggested making the lack of deprecation warnings explicit. Are you OK with this simplification?

Reply:
> because there is no commitment given for this in the main contributing guidelines
But we still promise customers that we don't make braking changes to standard libraries, right?

Reply:
I understand that this sentence is implicitly in contrast to the standard libraries, but it doesn't actually say so. It just warns users that these queries are excempt from any stability policy that either (1) may one day be written down for the standard libraries or (2) may be assumed to exist based on the maturity of our software.

I think the suggested simplification ("are not actively maintained") is too dire and goes against the purpose of the `experimental` dir, which is to accept contributions into a directory that doesn't just look like the Recycle Bin. I think the original version had just the right balance between avoiding a maintenance burden for ourselves and still acknowledging that the accepted queries are valuable.

Reply:
@felicitymay, if you'd like to break the sentence down into shorter ones, how about:

```suggestion
Experimental queries and libraries may not be actively maintained as the standard libraries evolve. They may also be changed in backwards-incompatible ways or may be removed entirely in the future without deprecation warnings.
```

Reply:
Thanks. @yo-h - your suggestion LGTM.

> But we still promise customers that we don't make breaking changes to standard libraries, right?

I still think that this ought to go somewhere in this repository - in part it explains why there's a high bar to adding queries/libraries to the core directories.

Reply:
> I still think that this ought to go somewhere in this repository - in part it explains why there's a high bar to adding queries/libraries to the core directories.

I've added a paragraph to the standard contribution guidelines in 63adc63.

Reply:
Thanks.